grep.yaml

```diff
@@ -41,10 +41,74 @@ update:
 
 test:
   pipeline:
-    - runs: |
+    - name: Version and help check
+      runs: |
         grep --version
         egrep --version
         egrep --help
         fgrep --version
         fgrep --help
         grep --help
```

Comment:
I realize this wasn't modified in your PR but a prompt to group commands in alphabetical order would make reading these easier. I initially thought we were missing `grep --help`.

Reply:
added to guidance.